### ISSUE TITLE
It needs extra Slash (/) for every page. Why?

### ISSUE THREAD

**Issue by chintakdholakia** [janithl/Kottu2012#3]
HI

In config.php, if i set up following : 
$config['basepath'] = 'http://indiaisrising.com';

No links work except homepage. I need to add extra slash in basepath like this:
$config['basepath'] = 'http://indiaisrising.com/';

But then every links open with extra slash in their URL. 
like : 
http://indiaisrising.com//admin
http://indiaisrising.com//all/tags/tech/
http://indiaisrising.com//m
http://indiaisrising.com//si/tags/tech/

Basically all the links open with extra slash and if i remove them, it gives me ERROR 404 NOT FOUND.

Please help!

**Comment by chintakdholakia** [comment]
I fixed it. The problem was in Config.php

/* Index of first argument */
$config['argstart']	= 2;

I changed it to : 

/* Index of first argument */
$config['argstart']	= 1;

and now it is fixed !!
